pkg/webui/lib/shared-messages.js

```diff
@@ -45,7 +45,7 @@ export default defineMessages({
   appEUI: 'AppEUI',
   appEUIJoinEUI: 'AppEUI/JoinEUI',
   appEUIDescription:
-    'The AppEUI is a global application ID that uniquely identifies the owner of the end ­device. AppEUI is used for end devices with LoRaWAN versions 1.0.0-1.0.3, JoinEUI for 1.0.4+.',
+    'The AppEUI uniquely identifies the owner of the end ­device. If no AppEUI is provided by your device manufacturer (usually for development), fill with zeros',
```

Comment:
```suggestion
    'The AppEUI uniquely identifies the owner of the end ­device. If no AppEUI is provided by your device manufacturer (usually for development), it can be filled with zeros.'
```

https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.9/DESIGN.md#general-considerations

> *   Do not use direct address of the user except for dialogs
> *   Do not use end-of-sentence periods, unless the text consists of multiple sentences